tests/testUtil.ts

```diff
@@ -67,6 +67,16 @@ export const createAPIGatewayEvent = ({
     return eventTemplate
 }
 
+export const createAPIGatewayProxyEvent = (args: any) => {
```

Comment:
do you think we could include the correct type for this param, rather than `any`?

Reply:
as im reusing that argument definition i move both interfaces into a new `types.ts` file at the same level